index.d.ts

```diff
@@ -227,12 +227,7 @@ export class Stripe {
 
 export interface PaymentCardTextFieldNativeEvent {
   valid: boolean;
-  params: {
-    number: string;
-    expMonth: string;
-    expYear: string;
-    cvc: string;
-  };
+  params: Pick<CardTokenParams, 'number' | 'expMonth' | 'expYear' | 'cvc'>;
```

Comment:
Fix is here for `onParamsChange`. The `expMonth: string;` and `expYear: string;` were wrong.